### PR TITLE
Interim PR for #13

### PR DESCRIPTION
**Pull request by MichalKononenko**
Fixing codeclimate violations will continue, but others need this code.

### CODE REVIEW

File: find_T1_model.py [hunk 1]
```diff
@@ -1,151 +0,0 @@
-"""
```
Comment (MichalKononenko) [inline find_T1_model.py:1]: I renamed this to t1_model.py

File: learn_T1_NMR.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
-model = find_T1_model()
```
Comment (vyshakhsanjeev) [inline learn_T1_NMR.py:24]: good work bro!
You're a real Brogrammer!

Comment (MichalKononenko) [inline learn_T1_NMR.py:24]: Thanks!

File: t1_model.py [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+        return cost
+        
+        
+        
+        
+        
+        
+        
+        
+
```
Comment (MichalKononenko) [inline t1_model.py]: Fix trailing newline

File: tests/test_t1_model.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+class TestFindT1Model(unittest.TestCase):
+
+    def setUp(self):
+        self.model = T1Model()
+
+    def test_is_instance(self):
+        """
+        Trivial test that gives coverage something to do
+        """
+        self.assertIsInstance(self.model, T1Model)
```
Comment (MichalKononenko) [inline tests/test_t1_model.py:19]: Add a trailing newline

**Pull request merged**